### PR TITLE
Splash Page and Cypress Testing

### PR DESCRIPTION
**Pull request by JeffSalinas**
Added a global theme with the context api and material-ui hooks. Installed Cypress and wrote tests for the splash page.

### CODE REVIEW

File: cypress/integration/examples/render.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    cy.get('#testRequestButton').click()
+    cy.url().should('match', /request/)
+  })
+
+  it('should reload the request page and still have content served', function() {
+    cy.reload()
+    cy.url().should('match', /request/)
+    cy.get('#requestTest').should('exist')
+  } )
+})
```
Comment (juliakim) [inline cypress/integration/examples/render.js:60]: Add semicolons please

File: cypress/integration/examples/render.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  })
+
+  it('Should open menu on availability page', function () {
+    cy.visit('/availability');
+    cy.get('#testMenuAvailability').click()
+    cy.window().its('testOpen').should('equal', true)
+  })
+  
+
+
```
Comment (juliakim) [inline cypress/integration/examples/render.js]: Remove spaces

File: cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (juliakim) [inline cypress/support/commands.js:1]: Did you leave this in here for me? Or did you mean to add this to the gitignore?

File: dist/style.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.menuIcon {
+  position: relative;
+  bottom: 12px;
+  z-index: -1;
+}
+
+h1 {
+  font-family: Helvetica Neue, Helvetica;
```
Comment (juliakim) [inline dist/style.css]: Perhaps move font-family to body if you plan on using the same font throughout

File: server/server.js [hunk 5]
```diff
@@ -1,7 +1,17 @@
 const express = require('express');
+const path = require('path')
```
Comment (juliakim) [inline server/server.js]: ;

File: src/GlobalTheme.jsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+  whiteButton: {
+    background: '#F4F4F5',
+    border: 0,
+    borderRadius: 20,
+    boxShadow: '0 0 0 0 #000000',
+    color: 'black',
+    height: 48,
+    padding: '0 30px',
+    width: 345,
+    height: 63,
```
Comment (juliakim) [inline src/GlobalTheme.jsx]: Remove first height

File: src/GlobalTheme.jsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    fontSize: 24,
+    fontWeight: "bold",
+    fontFamily: 'Helvetica',
```
Comment (juliakim) [inline src/GlobalTheme.jsx:17]: If you want, can use shorthand for font

File: src/GlobalTheme.jsx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+  loginButton: {
+    background: '#EA882C',
+    border: 0,
+    borderRadius: 20,
+    boxShadow: '0 4px 4px 0 #000000',
+    color: 'black',
+    height: 48,
+    padding: '0 30px',
+    width: 345,
+    height: 63,
```
Comment (juliakim) [inline src/GlobalTheme.jsx]: Repead height

File: src/GlobalTheme.jsx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    fontSize: 24,
+    fontWeight: "bold",
+    fontFamily: 'Helvetica',
```
Comment (juliakim) [inline src/GlobalTheme.jsx:31]: Shorthand?

File: src/components/AdminSplash.jsx [hunk 10]
```diff
@@ -0,0 +1,57 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { withRouter, NavLink, useLocation } from 'react-router-dom';
+import Fab from '@material-ui/core/Fab';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import gobalTheme from '../ThemeContext.js';
```
Comment (juliakim) [inline src/components/AdminSplash.jsx]: Spelling

File: src/components/AdminSplash.jsx [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+import Menu from './Menu.jsx'
+
+const AdminSplash = () => {
+  const tempClasses = ['MCSP 02', 'MCSP 03', 'HRATX 44', 'HRATX 45'];
+  const [classes, setClasses] = useState(tempClasses)
+  const theme = useContext(gobalTheme);
```
Comment (juliakim) [inline src/components/AdminSplash.jsx]: globalTheme?

File: src/components/Availability.jsx [hunk 12]
```diff
@@ -0,0 +1,50 @@
+import React, { useState, useContext } from 'react';
+import { withRouter, useHistory } from 'react-router-dom';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import gobalTheme from '../ThemeContext.js';
```
Comment (juliakim) [inline src/components/Availability.jsx]: Spelling

**Review by juliakim — CHANGES_REQUESTED** [review]
Can you make pull requests smaller please lol

File: cypress/integration/examples/render.js (hunk 1 above)
Comment (JeffSalinas) [reply cypress/integration/examples/render.js:60]: No semicolons in cypress / mocha / chai

File: cypress/support/commands.js (hunk 3 above)
Comment (JeffSalinas) [reply cypress/support/commands.js:1]: The is boilerplate code that comes when you install cypress

**Pull request closed without merging**